js/renovation/viz/sparklines/utils.ts

```diff
@@ -29,3 +41,40 @@ export const createAxis = (isHorizontal: boolean): Axis => {
     getMarginOptions: (): {} => ({}),
   };
 };
+
+const generateDefaultCustomizeTooltipCallback = (
+  fontOptions?: Font, rtlEnabled?: boolean,
+): CustomizeTooltipFn => {
+  const { lineSpacing, size } = fontOptions ?? { };
+  const lineHeight = (lineSpacing ?? DEFAULT_LINE_SPACING) + (size ?? 0);
+
+  return (customizeObject: SparklineTooltipData): CustomizedOptions => {
+    let html = '';
+    const vt = customizeObject.valueTexts || [];
+    for (let i = 0; i < vt.length; i += 2) {
+      html += `<tr><td>${vt[i]}</td><td style='width: 15px'></td><td style='text-align: ${rtlEnabled ? 'left' : 'right'}'>${vt[i + 1]}</td></tr>`;
+    }
+
+    return { html: `<table style='border-spacing:0px; line-height: ${lineHeight}px'>${html}</table>` };
+  };
+};
+
+export const generateCustomizeTooltipCallback = (
+  customizeTooltip?: CustomizeTooltipFn,
+  fontOptions?: Font,
+  rtlEnabled?: boolean,
+): CustomizeTooltipFn => {
+  const defaultCustomizeTooltip = generateDefaultCustomizeTooltipCallback(fontOptions, rtlEnabled);
+
+  if (customizeTooltip && isFunction(customizeTooltip)) {
```

Comment:
Similar to isFunction()

Reply:
Ok

Reply:
linter says it's a bad idea to remove this check because customizeTooltip can be undefined (ESLint doesn't know what isFunction checking)

Reply:
https://trello.com/c/oSnG9nzt/1360-remove-undefined-operator-from-tooltip-props